### PR TITLE
Auto load data for readme

### PR DESCRIPTION
**Pull request by bhadurishirsesh**
**Description**
**Updated**
Following conversation on conclusions from scaled sklearn data, data source has been updated to URL of original dataset at https://www4.stat.ncsu.edu/~boos/var.select/diabetes.tab.txt 
- Related changes in conclusions for BMI have been reverted with real values of BMI
- Notebook has been updated to align with Readme

**Original PR**
Readme and example notebook use a CSV file that was a part of the repository. This dataset is available with sklearn and this change automates dataload from sklearn instead of CSV. Key changes and description of commits are:
- **Loading data from sklearn instead of csv in readme:**  This commit removes read csv step and replaces it with loading bunch object from sklearn.datasets. To enhance readbility, column names of the native dataset have been changed too
- **Fixing what looks like typos in conclusions for synergy:** Conclusion of TC having synergistic relationship with other variables looked incorrect and misaligned. This has been updated to align with the visualisation
- **Changing column name from TSH to TCH:**  This is a change made in the markdowns; sklearn documentation shows TCH would be a better description of this column than TSH
- **Remove part of conclusion in simulation for BMI:** sklearn data is scaled, and therefore conclusion based on raw value of BMI is harder to interpret for the user. Since, the actual value of the conclusion is less important that the demo of the functionality, the part of the conclusion referring to raw value of BMI has been removed.
- **Updating diabetes notebook to align with readme:** Changes in Readme are replicated in the notebook in sphinx folder
- **Deleting diabetes.csv:** Since, the example does not need the csv anymore, the file has been removed

**How to test**
- Open notebook (this can be in your local) and use the code in updated readme
- The conclusions in readme, should be aligned with the results you see in your notebook

### CODE REVIEW

**Review by j-ittner — DISMISSED** [review]
Looks great, thanks!

**Comment by j-ittner** [comment]
@bhadurishirsesh I noticed that the simulation output no longer shows true BMI numbers, instead these numbers appear to be centered/normalized. Any thoughts?

**Comment by bhadurishirsesh** [comment]
> @bhadurishirsesh I noticed that the simulation output no longer shows true BMI numbers, instead these numbers appear to be centered/normalized. Any thoughts?

@j-ittner  - the sklearn data set is scaled unlike the csv which had raw values - this is the reason why I had the commit fd13602 where the conclusion was edited to not refer to the true BMI values. My rationale is that the demo of the functionality is more important than the true BMI value. 

How the sklearn data is scaled (copied from sklearn documentation for sklearn.datasets.load_diabetes:
Each of these 10 feature variables have been mean centered and scaled by the standard deviation times n_samples (i.e. the sum of squares of each column totals 1).

**Comment by j-ittner** [comment]
Got it, thanks. I think the example is stronger with "real" values: it is good to talk about real BMI numbers to give a sense of where the "risky" area may begin. So the options are: (1) keep using the local csv, or (2) find a different public source
Thoughts?

**Comment by bhadurishirsesh** [comment]
> Got it, thanks. I think the example is stronger with "real" values: it is good to talk about real BMI numbers to give a sense of where the "risky" area may begin. So the options are: (1) keep using the local csv, or (2) find a different public source Thoughts?

Makes sense - I have found another public data source that has real BMI values (and real values for all other features), will test if the output is same/similar and make changes after testing it.

**Review by j-ittner — APPROVED** [review]
Thanks for updating, looking great!

**Comment by j-ittner** [comment]
Closes #308
